types/looks-same/looks-same-tests.ts

```diff
@@ -0,0 +1,56 @@
+import * as looksSame from 'looks-same';
```

Comment:
`looksSame` is a callable or constructable entity, it isn't legal in ES2015 to import this with a namespace import like `import * as foo from 'foo'`, so could you please convert this to either
```js
import foo = require('foo')
```
or the following with esModuleInterop turned on
```js
import foo from 'foo';
```